packages/shared/src/hooks/feed/useTrackImpression.ts

```diff
@@ -42,10 +39,6 @@ export default function useTrackImpression(
               ...feedAnalyticsExtra(feedName, ranking, {
                 scroll_y: window.scrollY,
               }).extra,
-              feedback:
-                isFeedbackEnabled && item.post.type === PostType.Article
-                  ? true
-                  : undefined,
```

Comment:
@idoshamun Same here, waiting for [this](https://dailydotdev.slack.com/archives/C04UN615UAF/p1698388838610749).